src/chrome/content/rules/administrator.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="administrator.de">
+<target host="*.administrator.de" />
```

Comment:
can you confirm there is any subdomain (aside from www.)? I don't see any in my testing.